presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -95,3 +95,13 @@ Array Functions
 
     Subsets array ``x`` starting from index ``start`` (or starting from the end
     if ``start`` is negative) with a length of ``length``.
+
+.. function:: zip(array1, array2[, array3[, array4]]) -> array<row>
+
+    | Aggregate elements of the arrays passed as arguments into rows. Similar to the combined use of ``UNNEST`` and ``ARRAY_AGG``.
+    | The M-th element of the N-th argument will be the N-th field of the M-th output element.
+    | If the arguments have uneven length, missing values are filled with NULL. Null arguments are treated like empty arrays.
```

Comment:
Treating ``NULL`` arguments as empty arrays seems unintuitive. Most SQL functions return ``NULL`` whenever an argument is ``NULL``, so I would change this behavior. If the user wants that, they can always do ``zip(coalesce(a, array[]), coalesce(b, array[]))``